src/components/body/navigation.css

```diff
@@ -55,7 +55,7 @@
 }
 
 .navigation-link.active {
-    color: #9a9c9e;
+    color: #30D5C8;
```

Comment:
Potentially try BBD9E7, it may be a bit more subtle but still distinguishes enough from the grey

Reply:
Yeah we know garrett said so but i don't think the bright turquoise looks that good esthetically